src/CountryPicker.js

```diff
@@ -63,6 +63,7 @@ export default class CountryPicker extends Component {
     excludeCountries: React.PropTypes.array,
     styles: React.PropTypes.object,
     filterPlaceholder: React.PropTypes.string,
+    filterPlaceholderTextColor: React.PropTypes.string,
```

Comment:
Should this have a default value?

Reply:
I did think about that, but it wasn't clear what the default value should be?  Also I assume the default value could vary by platform.

I tested it on Android without a value and it reverted to what seemed to be the default.  I don't have an iOS environment set up to test it on right now.